src/Summoner/Template.hs

```diff
@@ -408,8 +417,8 @@ createStackTemplate ProjectData{..} = Dir (toString repo) $
 
         cache:
           directories:
-          - "$$HOME/.stack"
-          - "$$TRAVIS_BUILD_DIR/.stack-work"
+          $travisCabalCache
+          $travisStackCache
```

Comment:
Could you also, please, specify cache directories in `.travis.yml` file for `summoner` itself? Now there's only stack but `cabal` should be there as well.

Reply:
Do you mean `$TRAVIS_BUILD_DIR/dist-newstyle`?

Reply:
@willbasky No, I mean changing this file. Travis configs for `summoner` itself:

* https://github.com/kowainik/summoner/blob/master/.travis.yml